src/functional/ffront/common_types.py

```diff
@@ -10,6 +12,7 @@ class ScalarKind(IntEnum):
     BOOL = 1
     INT32 = 32
     INT64 = 64
+    INT = INT32 if np.int_ == np.int32 else INT64
```

Comment:
It could be worth to add a very short comment here to explain that this is needed because python `int`s are transformed to C `long` type by numpy, which may have different sizes on different platforms